### PR TITLE
Add details about supported service types, copy edits

### PR DESCRIPTION
**Pull request by mmachir**
- [x] Add details about supported service types for arc_open() and arc_select() per https://github.com/R-ArcGIS/planning/issues/1
- [x] Change "we" to "you"
- [x] Match objective order to tutorial order

### CODE REVIEW

File: location-services/read-data.qmd [hunk 1]
```diff
@@ -4,48 +4,59 @@ subtitle: "Learn how to read data from ArcGIS Online or Enterprise into R"
...
+The objective of this tutorial is to teach you how to:
 
-- read in a population dataset from ArcGIS Online
-- apply a filter to a Feature Layer
-- read only specified columns
-- find a Feature Layer url 
+- find a Feature Layer url from ArcGIS Online
+- read in the data from the Feature Layer
+- filter the Feature Layer data by column
+- filter the Feature Layer data by row
```
Comment (JosiahParry) [inline location-services/read-data.qmd:28]: I think these need to be rephrased. `filter` is a row-wise operation where we select rows based on some logical predicates and `select` is a column-wise operation where we fetch specific columns based on named 
```suggestion
- select the Feature Layer data by column
- filter the Feature Layer data by attributes
```

File: location-services/read-data.qmd [hunk 2]
```diff
@@ -4,48 +4,59 @@ subtitle: "Learn how to read data from ArcGIS Online or Enterprise into R"
...
+- use `dplyr` for filtering
```
Comment (JosiahParry) [inline location-services/read-data.qmd:29]:
```suggestion
- use `dplyr` for selecting and filtering
```

File: location-services/read-data.qmd [hunk 3]
```diff
@@ -55,54 +66,55 @@ Before we can read in the Feature Layer, we need to load the `arcgis` R package.
...
 ## Reading from a Feature Layer
 
-Once we have a `FeatureLayer` object we can read its data into memory using the `arc_select()` function. By default, if we use `arc_select()` on a `FeatureLayer` without any additional arguments, the entire service will be brought into memory.
+Once you have a `FeatureLayer` object, you can read its data into memory using the `arc_select()` function. By default, if you use `arc_select()` on a `FeatureLayer` without any additional arguments, the entire service will be brought into memory.
 
 :::{.callout-warning}
-Be careful to not try and read in more data than you need! Reading an entire feature services is fine for datasets in the realm of 0 - 5,000 features. But when we have more than 10,000 features performance and memory may be throttled. 
+Avoid reading in more data than you need! Reading an entire feature service is fine for datasets with fewer than 5,000 features. But when there are more than 10,000 features, performance and memory may be throttled. 
 
 Exceptionally detailed geometries require more data to be transferred across the web and may be slower to process.
```
Comment (JosiahParry) [inline location-services/read-data.qmd:97]:
```suggestion
Exceptionally detailed geometries require more data to be transferred across the web and may be slower to process and may require adjustment of the `page_size` argument of `arc_select()`.
```

**Review by JosiahParry — CHANGES_REQUESTED** [review]

File: location-services/read-data.qmd (hunk 1 above)
Comment (mmachir) [reply location-services/read-data.qmd]: Thank you for the fix! the phrasing I had was not sitting super well with me either.

**Pull request merged**